shell/src/main/java/alluxio/cli/bundler/command/CollectConfigCommand.java

```diff
@@ -52,10 +60,22 @@ public boolean hasSubCommand() {
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     mWorkingDirPath = getWorkingDirectory(cl);
-    String confDir = mFsContext.getClusterConf().get(PropertyKey.CONF_DIR);
+    String confDirPath = mFsContext.getClusterConf().get(PropertyKey.CONF_DIR);
 
-    // TODO(jiacheng): phase 2 copy intelligently, check security risks
-    FileUtils.copyDirectory(new File(confDir), new File(mWorkingDirPath), true);
+    File confDir = new File(confDirPath);
+    List<File> allFiles = CommonUtils.recursiveListLocalDir(confDir);
+    for (File f : allFiles) {
+      String filename = f.getName();
+      String relativePath = confDir.toURI().relativize(f.toURI()).getPath();
+      // Ignore file prefixes to exclude
+      for (String prefix : EXCLUDED_FILE_PREFIXES) {
+        if (filename.startsWith(prefix)) {
+          continue;
```

Comment:
Doesn't this `continue` just move to the next entry in `EXCLUDED_FILE_PREFIXES`? Don't we want to go to the next entry in `allFiles` instead?